File: 26-RemoveDuplicatesFromSortedArray/26-RemoveDuplicatesFromSortedArray.cpp
// Last updated: 07/04/2026, 11:42:57
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int k = 1 ;
        for (int i = 1 ; i<nums.size();i++){
            if (nums[i] !=nums[i-1]){
                nums[k] = nums[i];
                k++;
            }
        }
        return k ;
    }
};

File: 26-RemoveDuplicatesfromSortedArray/26-RemoveDuplicatesfromSortedArray.java
// Last updated: 08/04/2026, 10:10:42
1class Solution {
2    public int removeDuplicates(int[] nums) {
3        int k =1 ;
4        int n = nums.length;
5        for ( int i = 1;i<n;i++){
6            if ( nums[i-1] != nums[i]){
7                nums[k] = nums[i];
8                k++;
9            }
10        }
11        return k;
12
13        
14    }
15}